### PR TITLE
Changes to "it uses" section.

### PR DESCRIPTION
**Pull request by Pomax**
Closes https://github.com/mozilla/foundation.mozilla.org/issues/3823 by uplifting the device/app fields from NullBoolean to our new Yes/No/Unknown/NA field, and styling the infobox as per the comps.

Example product with all four values: https://foundation-mofostaging-pr-3855.herokuapp.com/en/privacynotincluded/products/percy-cypress/

### CODE REVIEW

File: network-api/networkapi/buyersguide/migrations/0032_uplift_app_device_fields.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# Generated by Django 2.2.5 on 2019-11-06 20:11
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/migrations/0032_uplift_app_device_fields.py]: similar to migration 0028, which did this for a whole bunch of other Boolean fields.

**Comment by beccaklam** [comment]
@Pomax This is looking good! 
Only major bug I found was that at the iPad view (both vertical and horizontal) the words are at risk of pushing into one another:
Screen Shot 2019-11-07 at 3 37 11 PM
I think perhaps we can start the mobile media query earlier at the iPad width?

**Comment by beccaklam** [comment]
> I would suggest removing `col-...` from the `.it-uses` text blocks and instead relying on flexbox a bit more for position + spacing. Can we try making the parent container `flex:column`, then `flex:row` + `justify-content:between` at `lg` screens?

That would be great! I did find the spacing a bit off but wasn't sure if there was a solution to it!

**Comment by Pomax** [comment]
updated the PR - `lg` was actually too small (several `unknown` would still overlap) so I've set it to columns for `xl` (1200+). If we want `lg` (992+) we probably need to fiddle with font sizes/condensed typeface/something shorter than "unknown"... =S

When using `lg`, I was seeing things like this (and that's not even all-unknowns):

image

**Pull request merged**